dataflow/flex-templates/streaming_beam/metadata.json

```diff
@@ -0,0 +1,31 @@
+
```

Comment:
Is this missing the initial `{` ?

Reply:
Thanks for catching it. Not sure how it got removed. Have fixed it.